Assets/Scripts/UI/Canvas/PhotoTaking/PhotoTaker.cs

```diff
@@ -1,12 +1,11 @@
 ﻿using System;
-using UnityEngine;
-using UnityEngine.UI;
-using UnityEngine.Events;
-
 using Framework.PhoneCamera;
 using Framework.ScriptableObjects;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.UI;
```

Comment:
```suggestion
using UnityEngine;
using UnityEngine.Events;
using UnityEngine.UI;

using Framework.PhoneCamera;
using Framework.ScriptableObjects;
```
Zelfde feedback als eerste 2 punten.